public/app/features/dashboard-scene/serialization/transformSaveModelSchemaV2ToScene.ts

```diff
@@ -179,7 +150,11 @@ export function transformSaveModelSchemaV2ToScene(dto: DashboardWithAccessInfo<D
     meta.canSave = false;
   }
 
-  const layoutManager: DashboardLayoutManager = createLayoutManager(dashboard);
+  const layoutManager: DashboardLayoutManager = layoutSerializerRegistry
+    .get(dashboard.layout.kind)
+    .serializer.deserialize(dashboard.layout, dashboard.elements, dashboard.preload);
```

Comment:
Can we add some error handling? Considering `layoutManager` is an important piece, we need to account for the case of `deserialize` throwing an error